### PR TITLE
[Crosswalk Lite] Add document and first release blog.

### PR DESCRIPTION
**Pull request by halton**

### CODE REVIEW

**Comment by halton** [comment]
@baleboy @bspencer could you review the CrosswalkMini document and blog?

Note: The release is not avialable on 01.org, this PR can be hold until it is available.

Question: Does the asset file be commit to `www/assets/` as well?

**Comment by halton** [comment]
CC CrosswalkMini memebers @wang16 @d0ngjun @lincsoon in case something wrong I written.

File: public/documentation/crosswalk_mini.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+</ol>
+
+### Embedded mode
+* Document: Same as https://crosswalk-project.org/documentation/getting_started.html
+* Download URL: https://download.01.org/crosswalk/releases/crosswalk/android/canary/VERSION/crosswalk-mini-VERSION.zip 
+
+### Cordova 
+* Document: Same as https://crosswalk-project.org/documentation/cordova.html
+* Download URL:
+ * x86: https://download.01.org/crosswalk/releases/crosswalk-mini/android/canary/VERSION/x86/crosswalk-cordova-VERSION-x86.zip
```
Comment (d0ngjun) [inline public/documentation/crosswalk_mini.md]: crosswalk-mini-cordova-VERION.zip

File: public/documentation/crosswalk_mini.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+ * arm: https://download.01.org/crosswalk/releases/crosswalk-mini/android/canary/VERSION/arm/crosswalk-cordova-VERSION-arm.zip
```
Comment (d0ngjun) [inline public/documentation/crosswalk_mini.md]: same as above.

**Comment by halton** [comment]
@baleboy @bspencer for your convenience, https://docs.google.com/a/intel.com/document/d/1cI4m1w8EO1_FUuwiDmYAgc_p-EYu_D6Bxxr4BnA5JvU/edit?usp=sharing is created for review.

**Comment by baleboy** [comment]
thanks @halton. let's continue the review and make the changes directly there, we can update the PR again once we are happy.

**Comment by halton** [comment]
@baleboy @bspencer @kenchris @alexshalamov I've updated the PR according the comments in google doc. There are two opens but I keep the content what @kenchris wrote:

> *Lite it hasn't been tested and validated by QA as the proper Crosswalk project. It is not recommended to use Crosswalk Lite for production at this stage. *.

I'm still unconformable to mention user that. Thus, user will be afraid to use that build, any better sentence?

> Long term, we would like to improve Chromium, Blink and Crosswalk to modularize its features, so that specific features (like WebRTC) can be turned on at the time the users build an APK.

I ever asked Alexis to talk with Googler when BlinkOn whether upsteram will accept those changes. I got answer is NO. In that case, should we mention that message here still?

**Comment by msrichmond** [comment]
Do not use the term "Crosswalk Lite" on the web site or anywhere in public.

Thanks!

MikeR

**Comment by halton** [comment]
@msrichmond @baleboy removed the word *Crosswalk Lite" from the doc. And the PR is updated accordingly, please review.

**Comment by halton** [comment]
@baleboy As request, the document is moved to wiki page: https://github.com/crosswalk-project/crosswalk-website/wiki/Crosswalk-Project-Lite. Only the blog thing left. (The screenshort image as well cause I do not find a way to upload image on github wiki).

**Comment by baleboy** [comment]
LGTM. The screenshot link in the wiki will need to be updated once this is published.

**Comment by halton** [comment]
@baleboy ready to merge and announce?

**Comment by baleboy** [comment]
yes. @bspencer ?

**Comment by halton** [comment]
@bspencer thanks for merging. I suppose you need do something to let https://crosswalk-project.org/blog/ have this change, correct?

**Comment by bspencer** [comment]
Yes.  Mike Shaver will push the change to production first thing tomorrow PST.  Sorry I got it checked in too late before he went home.
Bob

**Pull request merged**